src/java.base/share/classes/java/lang/Throwable.java

```diff
@@ -118,6 +119,11 @@ public class Throwable implements Serializable {
     @java.io.Serial
     private static final long serialVersionUID = -3042686055658047285L;
 
+    /**
+     * Flag that determines if exceptions should be traced by JFR
+     */
+    static volatile boolean tracing;
```

Comment:
The use-sites in Error check Throwable.tracing so not  immediately obvious that it's JFR tracing. Maybe it should be renamed to jfrTracing or an accessor of that name added so it's a bit clearer at the use-sites.

Reply:
Will change